### PR TITLE
points api endpoint reqs to wiki

### PR DESCRIPTION
**Pull request by tildeee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,334 +201,17 @@ Ideally, by the time of project submission, all smoke tests will pass.
 
 ## Functional Requirements
 
-Every endpoint must serve JSON data, and must use HTTP response codes to indicate the status of the request.
+All of the API requirements for this project are located in the project repo's wiki page.
```
Comment (CheezItMan) [inline README.md]: Just a suggested link

```suggestion
All of the API requirements for this project are located in the project repo's [wiki page](https://github.com/AdaGold/video-store-api/wiki).
```

**Review by CheezItMan — COMMENTED** [review]
Just a minor suggestion for convenience.  

Also are we using Postman like the README still states or are we using Cucumber?

File: README.md (hunk 1 above)
Comment (tildeee) [reply README.md]: @CheezItMan I love this! I'm under the assumption that the wikis get copied when you fork a repo, so I was wondering if it'd be more stable to always point them to their cohort wiki?

Comment (tildeee) [reply README.md]: Or does it not matter, and it's more important to point them to AdaGold's wiki?

Comment (tildeee) [reply README.md]: Ehhhh *checks clock* it's probably better to just go with this for now, but that's just an outstanding question I had!

**Pull request merged**